scripts/CodeCoverage.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import argparse
+import shutil
+import zipfile
+import ntpath
+import subprocess
+
+DEFAULT_WHITELIST = ".*cpp$"
+DEFAULT_BLACKLIST = "/usr/local/probe/.*"
+
+CMAKEFILEPATH = "CMakeLists.txt"
+LIBRARY_TO_BUILD_REGEX = 'SET\(LIBRARY_TO_BUILD (.*)\)$'
+
+def get_file_content(filename):
+    with open(filename) as f:
+        content = f.readlines()
+        return content
+
+def get_library_to_build_name(line_str):
+    match_obj = re.search(LIBRARY_TO_BUILD_REGEX, line_str, re.M|re.I)
+    if match_obj:
+        return True, match_obj.group(1)
+    return False, ""
```

Comment:
Nitpick, can be ignored -- someone might not immediately realize you're returning a tuple here ;) 

return (False, "")